mitodl/ocw-hugo-themes
# Fix table renders for course-v1 when table width is greater then main-content width

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #835 

#### What's this PR do?
- Removes console.log in course-info-toggle.js
- Fixes bug for course info buttons text
- Closes course info drawer if table width is greater than content width
- Reruns table width calculations on course info toggle

#### How should this be manually tested?
- Start ocw-course using course "RES.3-006_Sp21"
- Open a page with a table and verify the following
    - On pressing couse-info toggle table width is recalculated. ie table renders in mobile form when drawer is open and renders normally when drawer is closes
    - On page load drawer is closes

#### Screenshots (if appropriate)
(Optional)
![Screenshot 2022-09-14 at 5 06 38 PM](https://user-images.githubusercontent.com/87968618/190152101-ca928802-9ff6-404e-bdcf-ccaf64ddb592.png)
![Screenshot 2022-09-14 at 5 06 43 PM](https://user-images.githubusercontent.com/87968618/190152192-040c0e84-6b16-45aa-8381-95ed634dc046.png)
